Makefile

```diff
@@ -31,3 +31,45 @@ deep-copy:
 crds:
 	operator-sdk generate crds
 	mv ./deploy/crds/* ./charts/piraeus/crds
+
+release:
+	# check that VERSION is set
+	@if [ -z "$(VERSION)" ]; then \
+		echo "Usage: make prepare-release VERSION=<version>" >&2 ; \
+		exit 1 ; \
+	fi
+	# check that version has expected format
+	@if ! echo -e "$(VERSION)" | grep -cP '^\d+\.\d+\.\d+$$' > /dev/null ; then \
```

Comment:
using the count + /dev/null feels a bit uncommon. What abuot dropping `-c` and using `-q`?

Reply:
TIL `-q` :sweat_smile: